platform/core.network/src/org/netbeans/core/network/utils/LocalAddressUtils.java

```diff
@@ -92,29 +92,47 @@
     
     
     // 8.8.8.8  -- which incidentially is Google's DNS server, but we don't ever connect
-    private static final byte[] SOMEADDR_IPV4_RAW = new byte[]{
-        (byte)0x80, (byte)0x80, (byte)0x80, (byte)0x80, (byte)0x80, (byte)0x80, (byte)0x80, (byte)0x80};
+    private static final byte[] SOMEADDR_IPV4_RAW = new byte[]{0x08, 0x08, 0x08, 0x08};
 
```

Comment:
@lbruun @matthiasblaesing I changed this to fix the exception.

Reply:
@blackleg Thanks. It should of course have been a 4-byte array from the beginning. Too much cut'n paste. I think the correct solution from my part would have been to re-throw as a `RuntimeException` instead of a stupid remark "cannot happen by definition", which is strictly speaking true, but only if one doesn't screw up the array length. :-)

Reply:
@lbruun I changed this pull request to throw a RuntimeException.  Could you review this?